update-pks-team

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
This file looks useful during dev.

Consider giving a bit of comment header within each function.  

"update-permissions": consider "set-team-permissions"?

"get-credhub-credentials": this is a nice way to document that where the Director for concourse is living--I guess I'm asking to make the onramp easier for a new team member who might want to use this.